src/getReorderedGoods.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReorderOptions } from './Type/ReorderOptions';
+import { SortType } from './Type/SortType';
+
```

Comment:
this file should have `.ts` extension, it's just a helper and has nothing to do with react